jobs/env/ci-kubernetes-e2e-gci-gce-scalability.env

```diff
@@ -4,3 +4,6 @@
 
 # Override GCE defaults.
 CREATE_CUSTOM_NETWORK=true
+
+# For debugging schedule-to-run latency regression.
+KUBELET_TEST_LOG_LEVEL=--v=4
```

Comment:
This file is also used by our presubmit job - pull-kubernetes-e2e-gce-100-performance.
Let's not risk this change for that one. Could you instead set `--env=KUBELET_TEST_LOG_LEVEL=--v=4` for the job in config.json?

Reply:
Done.